test/functional/wallet_listreceivedby.py

```diff
@@ -172,7 +169,7 @@ def run_test(self):
         address = self.nodes[0].getnewaddress(label)
 
         reward = Decimal("25")
-        self.generatetoaddress(self.nodes[0], 1, address, sync_fun=self.no_op)
```

Comment:
What was the purpose of this in the first place? Was there any reason to have this?

Reply:
No idea. I think it was added in b5696750a925c07261287b043ffdfb393cbb1327. It is only needed for the last call to generate IIUC.